rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -466,11 +553,19 @@ def get_inference_input_dict(self, policy_id: PolicyID) -> \
         for view_col, view_req in view_reqs.items():
             # Create the batch of data from the different buffers.
             data_col = view_req.data_col or view_col
-            time_indices = \
-                view_req.shift - (
-                    1 if data_col in [SampleBatch.OBS, "t", "env_id",
-                                      SampleBatch.AGENT_INDEX] else 0)
+            delta = -1 if data_col in [
```

Comment:
Also more complex now to build an action input dict: Inputs could be over some range of time steps.